src/dcqc/file.py

```diff
@@ -112,7 +112,8 @@ def get_file_type(cls, file_type: str) -> FileType:
 FileType("CSV", (".csv"), "format_3752")
 FileType("BAM", (".bam"), "format_2572")
 FileType("FASTQ", (".fastq", ".fastq.gz", ".fq", ".fq.gz"), "format_1930")
-FileType("HDF5", (".hdf", ".hdf5", ".h5", ".he5", ".h5ad"), "format_3590")
+FileType("HDF5", (".hdf", ".hdf5", ".h5", ".he5"), "format_3590")
+FileType("H5AD", (".h5ad"), "format_3590")
```

Comment:
Ah, do we know if these files with these different extensions actually leverage different validation tools?  I wonder if that is worth asking the HTAN channel to understand what file extensions these H5AD files can come in